rules/turn-based-play/combat.md

```diff
@@ -23,6 +23,7 @@ Attacking comes in a few steps like so:
 ### Consequences
 Losing some components will have severe consequences for you.
 If one of your components being damaged/destroyed leaves you with too few resources to power your constant draw components, your mech will no longer function at all.
+A destroyed joint means you can no longer use attachments in that joint, and if the destroyed joint was your in your legs section you can no longer move.
```

Comment:
Line 26 and 27 may need to be swapped here? As read now, L27 seems to indicate that you can get joints working by disabling components

Reply:
Fixed